pom.xml

```diff
@@ -3,25 +3,30 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.1</version>
+        <version>2.14</version>
+        <relativePath/>
     </parent>
     <artifactId>mock-slave</artifactId>
     <version>1.10-SNAPSHOT</version>
     <packaging>hpi</packaging>
-    <name>Mock Slave Plugin</name>
+    <name>Mock Agent Plugin</name>
```

Comment:
Well of course it is not remote in the sense of network topology, but it is a true agent, using the `remoting` protocol; so it can be used to demonstrate/test code that is sensitive to that protocol, as well as code that is sensitive to node selection.